ui/app/components/ui/icon/index.scss

```diff
@@ -6,7 +6,7 @@
   }
 
   &--info {
-    fill: $info-blue;
```

Comment:
Nit: This variable is now unused, so it could be removed as well.

Reply:
It looks like i missed this line when trying to split my commits. it is removed in the next PR